db/migrate/20210308123731_add_discard_to_user_profiles.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AddDiscardToUserProfiles < ActiveRecord::Migration[6.1]
+  def change
+    add_column :lead_provider_profiles, :discarded_at, :datetime
+    add_index :lead_provider_profiles, :discarded_at
+
+    add_column :induction_coordinator_profiles, :discarded_at, :datetime
+    add_index :induction_coordinator_profiles, :discarded_at
+
+    add_column :early_career_teacher_profiles, :discarded_at, :datetime
+    add_index :early_career_teacher_profiles, :discarded_at
+
+    add_column :admin_profiles, :discarded_at, :datetime
+    add_index :admin_profiles, :discarded_at
```

Comment:
hmm, there is four of them here, and three everywhere else

Are we sure we want to add it to ect profiles in this PR?

Reply:
Added another migration reverting the changes